src/extractor/guidance/turn_discovery.cpp

```diff
@@ -41,7 +41,7 @@ bool findPreviousIntersection(const NodeID node_v,
      * PREVIOUS_ID. To verify that find, we check the intersection using our PREVIOUS_ID candidate
      * to check the intersection at NODE for via_edge
      */
-    const constexpr double COMBINE_DISTANCE_CUTOFF = 30;
+    const constexpr double COMBINE_WEIGHT_CUTOFF = 30;
```

Comment:
This should actually be `COMBINE_DISTANCE_CUTOFF`, looking at the coordinate code below.